scss/_spinners.scss

```diff
@@ -83,3 +83,11 @@
     }
   }
 }
+
+//
+// Modifiers
+//
+
+.spinner-reverse {
+  animation-direction: reverse;
```

Comment:
Wondering if it should be duplicated for each component using animation such as placeholders, progress bars, etc... Or if it should be a new utility 🤔

Reply:
I was wondering the same thing. Does the Bootstrap set of utilities need an `animation-direction` utility (or more globally `animation` utilities?
_Note: I haven't checked yet in the history whether this question has been raised before or not_

For the direction, it could be something like:
* `.animation-direction-normal`
* `.animation-direction-reverse`
* `.animation-direction-alternate`
* `.animation-direction-alternate-reverse`

If so, this PR could be dropped while waiting for this new utility.

But would work only for single animations (see https://developer.mozilla.org/en-US/docs/Web/CSS/animation-direction#syntax).

Thoughts @twbs/css-review?

Reply:
IMHO, the utility approach is way better here to fix #38053. I find `.animation-*` better.

Still need to determine if it is needed at Bootstrap level or at user level.

Reply:
I think having some transition and animation helpers or utilities would be great for our users.

Reply:
OK let's aim for that. I'll close this PR + https://github.com/twbs/bootstrap/pull/38054 and comment create a feature request in our backlog to summarize what's been said in both PRs.
And link this feature request to https://github.com/twbs/bootstrap/issues/38053.

Reply:
Done in https://github.com/twbs/bootstrap/issues/38150